src/api/artifact/abstractor/resolver/chart/chart.go

```diff
@@ -86,7 +90,44 @@ func (r *resolver) ResolveMetadata(ctx context.Context, manifest []byte, artifac
 }
 
 func (r *resolver) ResolveAddition(ctx context.Context, artifact *artifact.Artifact, addition string) (*resolv.Addition, error) {
-	// TODO implement
+	if addition != AdditionTypeValues {
```

Comment:
There are three additions for helm chart: `readme.md`, `values.yaml` and `dependencies`. We should return the separated one according to the request specified by `addition` parameter